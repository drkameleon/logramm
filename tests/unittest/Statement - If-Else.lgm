vadIs(x) => 
{ 
	      x==1 : out "is 1";
	else: x==2 : out "is 2";
	else: x==3 : out "is 3";
	else: x==4 : out "is 4";
	else: 
		out "is >=5!";
}

@ i->[1..20]: vadIs(i);