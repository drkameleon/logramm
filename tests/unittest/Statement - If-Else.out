is 1
is 2
is 3
is 4
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
is >=5!
